openfecwebapp/api_caller.py

```diff
@@ -9,11 +9,13 @@
 MAX_FINANCIALS_COUNT = 4
 
 
-def _call_api(path, filters):
+def _call_api(path, filters={}):
```

Comment:
It's unsafe to use mutable values as defaults, since one function call can change the value that the next call sees. How about a signature of `def _call_api(path, **filters)`?